numpy/ma/tests/test_core.py

```diff
@@ -2581,6 +2581,14 @@ def test_no_masked_nan_warnings(self):
             # also check that allclose uses ma ufuncs, to avoid warning
             allclose(m, 0.5)
 
+    def test_masked_array_underflow(self):
+        np.seterr(under="raise")
```

Comment:
Please make sure to use `with np.errstate()`, this will otherwise affect later tests randomly.  Otherwise, LGTM, no need to insert those newlines, but they seem fine.